assets/helpers/legal.js

```diff
@@ -3,21 +3,33 @@
 // ----- Imports ----- //
 
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { SubscriptionProduct } from './subscriptions';
 
 // ----- Terms & Conditions ----- //
 
-const defaultTermsLink: string = 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions';
+const defaultContributionTermsLink: string = 'https://www.theguardian.com/info/2016/apr/04/contribution-terms-and-conditions';
 
-const termsLinks: {
+const contributionsTermsLinks: {
   [CountryGroupId]: string,
 } = {
-  GBPCountries: defaultTermsLink,
+  GBPCountries: defaultContributionTermsLink,
   UnitedStates: 'https://www.theguardian.com/info/2016/apr/07/us-contribution-terms-and-conditions',
   AUDCountries: 'https://www.theguardian.com/info/2016/apr/08/australia-contribution-terms-and-conditions',
-  EURCountries: defaultTermsLink,
-  International: defaultTermsLink,
-  NZDCountries: defaultTermsLink,
-  Canada: defaultTermsLink,
+  EURCountries: defaultContributionTermsLink,
+  International: defaultContributionTermsLink,
+  NZDCountries: defaultContributionTermsLink,
+  Canada: defaultContributionTermsLink,
+};
+
+const subscriptionsTermsLinks: {
+  [SubscriptionProduct]: string,
+} = {
+  DigitalPack: 'https://www.theguardian.com/info/2014/aug/06/guardian-observer-digital-subscriptions-terms-conditions',
```

Comment:
We are only getting links for Digital Pack at the moment. I just like having a complete match.